src/uu/install/src/install.rs

```diff
@@ -529,13 +519,13 @@ fn standard(mut paths: Vec<String>, b: &Behavior) -> UResult<()> {
     if paths.is_empty() {
         return Err(UUsageError::new(
             1,
-            get_message("install-error-missing-file-operand"),
+            translate!("install-error-missing-file-operand", "util_name" => uucore::execution_phrase()),
```

Comment:
Is this intentional? Previously there were no arguments.
```suggestion
            translate!("install-error-missing-file-operand"),
```